tests/test_stories.py

```diff
@@ -9,57 +9,141 @@
+    'date': '2018-08-03',
     'description': 'Hot',
     'map_year': 2018,
     'x_coord': 50,
     'y_coord': 82
 }
 
-stories = [
-    {
-      '_id': 21,
-      'story_name': 'Angad Goes to Wisconsin',
-    },
-    {
-      '_id': 22,
-      'story_name': 'Amanda Takes her Medicine',
-    },
-    {
-      '_id': 23,
-      'story_name': 'Jeffy Goes to Fashion Show',
-    },
-    {
-      '_id': 24,
-      'story_name': 'Time for Hack4Impact',
-    }
-]
-            
-          
-    story = {
-    '_id': 25
+story_with_no_pois = {
     'story_name' : 'Jeffy Discovers the Dark side of the Moon'
 }
 
+story_with_pois = {
+    'story_name' : 'Angad Goes to Wisconsin',
+    'poi_ids' : []
+}
+
+story_edit = {
+      'story_name': 'Time for Hugs4Infants'
+}
+
+story_empty = {
+
+}
+
+
+os.environ['STORY_ID_1'] = '1' # this one doesn't have pois
+os.environ['STORY_ID_2'] = '2' # this one has the pois
+os.environ['POI_ID_1'] = '1'
+os.environ['POI_ID_2'] = '2'
+
+class POITests(unittest.TestCase):
+
+    def test1_post_story(self):
+        r = requests.post('http://127.0.0.1:5000/stories', json=story_with_no_pois)
+        response = r.json()
+        self.assertEqual(response['code'], 201)
+        self.assertEqual(response['result']['story']['story_name'], story_with_no_pois['story_name'])
+        os.environ['STORY_ID_1'] = str(response['result']['story']['_id'])
+
+    def test2_post_story_with_pois(self):
+        r = requests.post('http://127.0.0.1:5000/pois', json=poi1)
+        response = r.json()
+        story_with_pois['poi_ids'].append(response['result']['poi']['_id'])
+        os.environ['POI_ID_1'] = str(response['result']['poi']['_id'])
+        r = requests.post('http://127.0.0.1:5000/pois', json=poi2)
+        response = r.json()
+        story_with_pois['poi_ids'].append(response['result']['poi']['_id'])
+        os.environ['POI_ID_2'] = str(response['result']['poi']['_id'])
+        r = requests.post('http://127.0.0.1:5000/stories', json=story_with_pois)
+        response = r.json()
+        self.assertEqual(response['code'], 201)
+        self.assertEqual(response['result']['story']['story_name'], story_with_pois['story_name'])
+        os.environ['STORY_ID_2'] = str(response['result']['story']['_id'])
+
+    def test3_get_stories(self):
+        # GET with no parameters
+        story_id_1 = int(os.environ.get('STORY_ID_1'))
+        story_id_2 = int(os.environ.get('STORY_ID_2'))
+        r = requests.get('http://127.0.0.1:5000/stories')
+        response = r.json()
+        self.assertEqual(response['code'], 200)
+        story = next((s for s in response['result']['stories'] if s['_id'] == story_id_1), None)
+        self.assertEqual(story['story_name'], story_with_no_pois['story_name'])
+        story = next((s for s in response['result']['stories'] if s['_id'] == story_id_2), None)
+        self.assertEqual(story['story_name'], story_with_pois['story_name'])
+
+    def test4_get_stories(self):

```

Comment:
since this request should "Get [a] list of stories a POI is in if given poi_id", what we want to test here is whether the correct story ids exist within a poi that's in that story, _and_ check that stories that the poi isn't in doesn't show up. so you should get one of the pois and check that its stories list contains `story_id_2` and does not contain `story_id_1`